core/src/main/scala/kafka/server/BrokerLifecycleManager.scala

```diff
@@ -410,6 +444,7 @@ class BrokerLifecycleManager(
         val errorCode = Errors.forCode(message.data().errorCode())
         if (errorCode == Errors.NONE) {
           failedAttempts = 0
+          eventQueue.append(new OfflineDirsClearEvent(dirsInFlight))
```

Comment:
Do we always want to queue an event even if dirsInFlight is empty?

Reply:
Good point. I missed that!